qiskit/qobj/converters/pulse_instruction.py

```diff
@@ -545,6 +564,31 @@ def gen_sf_schedule(*args, **kwargs):
 
         return instructions.SetFrequency(frequency, channel) << t0
 
+    @bind_name('shiftf')
+    def convert_shift_frequency(self, instruction):
+        """Return converted `ShiftFrequency`.
+
+        Args:
+            instruction (PulseQobjInstruction): Shift frequency qobj instruction.
+
+        Returns:
+            Schedule: Converted and scheduled Instruction
+        """
+        t0 = instruction.t0
+        channel = self.get_channel(instruction.ch)
+        frequency = instruction.frequency
```

Comment:
Same here regarding frequency. The implementer of `SetFrequency` also appears to have made the same mistake? This should be clarified in the Qobj spec too.

Reply:
I will open another PR to fix this bug.